mackup/applications/t.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = t
+
+[configuration_files]
+.trc
```

Comment:
It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

Reply:
Thanks, fixed. Did the same for the other two PRs.